python/cudnn/gemm_srelu/api.py

```
@@ -71,32 +80,27 @@ def __init__(
 
-        self._interpret_uint8_as_fp4x2 = True
-        self._kernel = Sm100BlockScaledPersistentDenseGemmKernel
-
     def check_support(self) -> bool:
-        m, k, l = self._tensor_shape(self.a_desc, name="sample_a")
-        n, b_k, b_l = self._tensor_shape(self.b_desc, name="sample_b")
-        c_m, c_n, c_l = self._tensor_shape(self.c_desc, name="sample_c")
-        d_m, d_n, d_l = self._tensor_shape(self.d_desc, name="sample_d")
-
-        self._value_error_if((b_k, b_l) != (k, l), f"B shape mismatch: expected (*, {k}, {l}), got {(n, b_k, b_l)}")
+        m, n, k, l = require_gemm_shapes(
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)